src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -34,6 +34,17 @@
  */
 class QuestionHelper extends Helper
 {
+    public const KEY_ARROW_RIGHT = "\033[C";
```

Comment:
This constants must be private. They are internal values that we don't need to call the class.

For tests, it would be fine to duplicate the constants.

Reply:
Yes, that's better.